src/main/java/org/schemaspy/mutation/ImpliedConstraint.java

```diff
@@ -0,0 +1,121 @@
+package org.schemaspy.mutation;
+
+import org.schemaspy.model.DatabaseObject;
+import org.schemaspy.model.ImpliedForeignKeyConstraint;
+import org.schemaspy.model.Table;
+import org.schemaspy.model.TableColumn;
+
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+/**
+ * Search supplied Collection of tables and add ImpliedForeignKeyConstraints
+ * when a childColumn doesn't have a parent and finds a primary key that
+ * matches based on name and type.
+ */
+
+public class ImpliedConstraint implements Mutation<ImpliedForeignKeyConstraint> {
+
+    // getting implied constraints has a side-effect of associating the parent/child tables, so don't do it
+    // here unless they want that behavior
```

Comment:
Misplaced comment

Reply:
Misplaced or redundant, should be removed.